Cannot find module 'postcss-bidirection'

Using instructions within Getting Started article on Windows 10, when I run "yarn start", it breaks as follows. Is this related to #7941?

PS C:\debugger> yarn start
yarn run v1.13.0
$ node bin/dev-server
Listening for WS on localhost:8116, all traffic is proxied to localhost:6080
Protocol messages can be logged by enabling `logging.firefoxProxy` `in /configs/local.json`
Hot Reloading - https://github.com/firefox-devtools/debugger.html/blob/master/docs/local-development.md#hot-reloading
View debugger examples here:
https://github.com/firefox-devtools/debugger-examples
Development Server Listening at http://localhost:8000

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-contextmenu/menu.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/devtools-contextmenu/menu.css 4:14-101
 @ ./node_modules/devtools-contextmenu/menu.js
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/QuickOpenModal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/QuickOpenModal.css 4:14-143
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Footer.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Footer.css 4:14-141
 @ ./src/components/Editor/Footer.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Tabs.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Tabs.css 4:14-139
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/AccessibleImage.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/AccessibleImage.css 4:14-150
 @ ./src/components/shared/AccessibleImage.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Dropdown.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Dropdown.css 4:14-143
 @ ./src/components/shared/Dropdown.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/CommandBar.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/CommandBar.css 4:14-145
 @ ./src/components/SecondaryPanes/CommandBar.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/ResultList.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/ResultList.css 4:14-145
 @ ./src/components/shared/ResultList.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/App.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/App.css 4:14-132
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/SearchInput.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/SearchInput.css 4:14-146
 @ ./src/components/shared/SearchInput.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Modal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Modal.css 4:14-140
 @ ./src/components/shared/Modal.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/variables.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/variables.css 4:14-138
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/WelcomeBox.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/WelcomeBox.css 4:14-139
 @ ./src/components/WelcomeBox.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/reps.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/reps.css 4:14-139
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/SourceIcon.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/SourceIcon.css 4:14-145
 @ ./src/components/shared/SourceIcon.js
 @ ./src/components/Editor/Tab.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/menu.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/menu.css 4:14-139
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/ProjectSearch.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/ProjectSearch.css 4:14-142
 @ ./src/components/ProjectSearch.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/ShortcutsModal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/ShortcutsModal.css 4:14-143
 @ ./src/components/ShortcutsModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/SecondaryPanes.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/SecondaryPanes.css 4:14-149
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Breakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Breakpoints.css 4:14-146
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Editor.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Editor.css 4:14-141
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-mc-assets/assets/devtools/client/themes/dark-theme.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/devtools-mc-assets/assets/devtools/client/themes/dark-theme.css 4:14-131
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-mc-assets/assets/devtools/client/themes/light-theme.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/devtools-mc-assets/assets/devtools/client/themes/light-theme.css 4:14-132
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/CommandBarButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Button/styles/CommandBarButton.css 4:14-163
 @ ./src/components/shared/Button/CommandBarButton.js
 @ ./src/components/SecondaryPanes/CommandBar.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/Sources.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/PrimaryPanes/Sources.css 4:14-142
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/utils/editor/source-editor.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/utils/editor/source-editor.css 4:14-148
 @ ./src/utils/editor/source-editor.js
 @ ./src/utils/editor/create-editor.js
 @ ./src/utils/editor/index.js
 @ ./src/utils/source.js
 @ ./src/workers/pretty-print/index.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Scopes.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Scopes.css 4:14-141
 @ ./src/components/SecondaryPanes/Scopes.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Highlight.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Highlight.css 4:14-144
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/EventListeners.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/EventListeners.css 4:14-149
 @ ./src/components/SecondaryPanes/EventListeners.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Workers.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Workers.css 4:14-142
 @ ./src/components/SecondaryPanes/Workers.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Accordion.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Accordion.css 4:14-144
 @ ./src/components/shared/Accordion.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/XHRBreakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/XHRBreakpoints.css 4:14-149
 @ ./src/components/SecondaryPanes/XHRBreakpoints.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/PaneToggleButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Button/styles/PaneToggleButton.css 4:14-163
 @ ./src/components/shared/Button/PaneToggleButton.js
 @ ./src/components/shared/Button/index.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/ConditionalPanel.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/ConditionalPanel.css 4:14-151
 @ ./src/components/Editor/ConditionalPanel.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/CloseButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Button/styles/CloseButton.css 4:14-158
 @ ./src/components/shared/Button/CloseButton.js
 @ ./src/components/shared/Button/index.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/codemirror/lib/codemirror.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/codemirror/lib/codemirror.css 4:14-113
 @ ./src/utils/editor/source-editor.js
 @ ./src/utils/editor/create-editor.js
 @ ./src/utils/editor/index.js
 @ ./src/utils/source.js
 @ ./src/workers/pretty-print/index.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/Outline.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/PrimaryPanes/Outline.css 4:14-142
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./packages/devtools-splitter/src/SplitBox.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./packages/devtools-splitter/src/SplitBox.css 4:14-143
 @ ./packages/devtools-splitter/src/SplitBox.js
 @ ./packages/devtools-splitter/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/SearchBar.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/SearchBar.css 4:14-144
 @ ./src/components/Editor/SearchBar.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/PreviewFunction.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/PreviewFunction.css 4:14-150
 @ ./src/components/shared/PreviewFunction.js
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/ManagedTree.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/ManagedTree.css 4:14-146
 @ ./src/components/shared/ManagedTree.js
 @ ./src/components/PrimaryPanes/SourcesTree.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/OutlineFilter.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/PrimaryPanes/OutlineFilter.css 4:14-148
 @ ./src/components/PrimaryPanes/OutlineFilter.js
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/A11yIntention.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/A11yIntention.css 4:14-142
 @ ./src/components/A11yIntention.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Expressions.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Expressions.css 4:14-146
 @ ./src/components/SecondaryPanes/Expressions.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Breakpoints/Breakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Breakpoints/Breakpoints.css 4:14-152
 @ ./src/components/SecondaryPanes/Breakpoints/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Preview/Popup.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/Editor/Preview/Popup.css 4:14-146
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/react-aria-components/src/tabs/tab.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/react-aria-components/src/tabs/tab.css 4:14-112
 @ ./node_modules/react-aria-components/src/tabs/tab.js
 @ ./node_modules/react-aria-components/src/tabs/index.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/react-aria-components/src/tabs/tab-list.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./node_modules/react-aria-components/src/tabs/tab-list.css 4:14-117
 @ ./node_modules/react-aria-components/src/tabs/tab-list.js
 @ ./node_modules/react-aria-components/src/tabs/index.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/Frames.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/Frames.css 4:14-147
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Popover.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Popover.css 4:14-142
 @ ./src/components/shared/Popover.js
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/Group.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/Group.css 4:14-146
 @ ./src/components/SecondaryPanes/Frames/Group.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/WhyPaused.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/WhyPaused.css 4:14-150
 @ ./src/components/SecondaryPanes/Frames/WhyPaused.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/BracketArrow.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/BracketArrow.css 4:14-147
 @ ./src/components/shared/BracketArrow.js
 @ ./src/components/shared/Popover.js
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Badge.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at module.exports (C:\debugger\postcss.config.js:37:7)
    at config.load.then (C:\debugger\node_modules\postcss-load-config\src\index.js:62:18)
 @ ./src/components/shared/Badge.css 4:14-140
 @ ./src/components/shared/Badge.js
 @ ./src/components/SecondaryPanes/Frames/Group.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js
webpack: Failed to compile.

I just ran into this problem on Ubuntu 16.04 while running `yarn start`. It occurs when running `yarn start` in the master branch of my local fork of debugger, which is currently even with `firefox-devtools:master`.

Below is the output from my `yarn start`:

```
yarn run v1.13.0
$ node bin/dev-server
Listening for WS on localhost:8116, all traffic is proxied to localhost:6080
Protocol messages can be logged by enabling `logging.firefoxProxy` `in /configs/local.json`
Hot Reloading - https://github.com/firefox-devtools/debugger.html/blob/master/docs/local-development.md#hot-reloading
View debugger examples here:
https://github.com/firefox-devtools/debugger-examples
Development Server Listening at http://localhost:8000

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/Sources.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/PrimaryPanes/Sources.css 4:14-142
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-contextmenu/menu.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/devtools-contextmenu/menu.css 4:14-101
 @ ./node_modules/devtools-contextmenu/menu.js
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/WelcomeBox.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/WelcomeBox.css 4:14-139
 @ ./src/components/WelcomeBox.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Footer.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Footer.css 4:14-141
 @ ./src/components/Editor/Footer.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Tabs.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Tabs.css 4:14-139
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/AccessibleImage.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/AccessibleImage.css 4:14-150
 @ ./src/components/shared/AccessibleImage.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Dropdown.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Dropdown.css 4:14-143
 @ ./src/components/shared/Dropdown.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/CommandBar.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/CommandBar.css 4:14-145
 @ ./src/components/SecondaryPanes/CommandBar.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/SearchInput.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/SearchInput.css 4:14-146
 @ ./src/components/shared/SearchInput.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Modal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Modal.css 4:14-140
 @ ./src/components/shared/Modal.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/App.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/App.css 4:14-132
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/ResultList.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/ResultList.css 4:14-145
 @ ./src/components/shared/ResultList.js
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/SourceIcon.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/SourceIcon.css 4:14-145
 @ ./src/components/shared/SourceIcon.js
 @ ./src/components/Editor/Tab.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/reps.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/reps.css 4:14-139
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/menu.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/menu.css 4:14-139
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/ProjectSearch.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/ProjectSearch.css 4:14-142
 @ ./src/components/ProjectSearch.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-mc-assets/assets/devtools/client/themes/dark-theme.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/devtools-mc-assets/assets/devtools/client/themes/dark-theme.css 4:14-131
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/CommandBarButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Button/styles/CommandBarButton.css 4:14-163
 @ ./src/components/shared/Button/CommandBarButton.js
 @ ./src/components/SecondaryPanes/CommandBar.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/utils/editor/source-editor.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/utils/editor/source-editor.css 4:14-148
 @ ./src/utils/editor/source-editor.js
 @ ./src/utils/editor/create-editor.js
 @ ./src/utils/editor/index.js
 @ ./src/utils/source.js
 @ ./src/workers/pretty-print/index.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/SecondaryPanes.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/SecondaryPanes.css 4:14-149
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/PaneToggleButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Button/styles/PaneToggleButton.css 4:14-163
 @ ./src/components/shared/Button/PaneToggleButton.js
 @ ./src/components/shared/Button/index.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Breakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Breakpoints.css 4:14-146
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Editor.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Editor.css 4:14-141
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Highlight.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Highlight.css 4:14-144
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Button/styles/CloseButton.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Button/styles/CloseButton.css 4:14-158
 @ ./src/components/shared/Button/CloseButton.js
 @ ./src/components/shared/Button/index.js
 @ ./src/components/Editor/Tabs.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/ManagedTree.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/ManagedTree.css 4:14-146
 @ ./src/components/shared/ManagedTree.js
 @ ./src/components/ProjectSearch.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/devtools-mc-assets/assets/devtools/client/themes/light-theme.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/devtools-mc-assets/assets/devtools/client/themes/light-theme.css 4:14-132
 @ ./node_modules/devtools-launchpad/src/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/QuickOpenModal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/QuickOpenModal.css 4:14-143
 @ ./src/components/QuickOpenModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/ConditionalPanel.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/ConditionalPanel.css 4:14-151
 @ ./src/components/Editor/ConditionalPanel.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/SearchBar.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/SearchBar.css 4:14-144
 @ ./src/components/Editor/SearchBar.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/Outline.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/PrimaryPanes/Outline.css 4:14-142
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Scopes.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Scopes.css 4:14-141
 @ ./src/components/SecondaryPanes/Scopes.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/EventListeners.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/EventListeners.css 4:14-149
 @ ./src/components/SecondaryPanes/EventListeners.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Accordion.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Accordion.css 4:14-144
 @ ./src/components/shared/Accordion.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Workers.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Workers.css 4:14-142
 @ ./src/components/SecondaryPanes/Workers.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/XHRBreakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/XHRBreakpoints.css 4:14-149
 @ ./src/components/SecondaryPanes/XHRBreakpoints.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/variables.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/variables.css 4:14-138
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./packages/devtools-splitter/src/SplitBox.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./packages/devtools-splitter/src/SplitBox.css 4:14-143
 @ ./packages/devtools-splitter/src/SplitBox.js
 @ ./packages/devtools-splitter/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Expressions.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Expressions.css 4:14-146
 @ ./src/components/SecondaryPanes/Expressions.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/PreviewFunction.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/PreviewFunction.css 4:14-150
 @ ./src/components/shared/PreviewFunction.js
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/codemirror/lib/codemirror.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/codemirror/lib/codemirror.css 4:14-113
 @ ./src/utils/editor/source-editor.js
 @ ./src/utils/editor/create-editor.js
 @ ./src/utils/editor/index.js
 @ ./src/utils/source.js
 @ ./src/workers/pretty-print/index.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/PrimaryPanes/OutlineFilter.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/PrimaryPanes/OutlineFilter.css 4:14-148
 @ ./src/components/PrimaryPanes/OutlineFilter.js
 @ ./src/components/PrimaryPanes/Outline.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/A11yIntention.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/A11yIntention.css 4:14-142
 @ ./src/components/A11yIntention.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/ShortcutsModal.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/ShortcutsModal.css 4:14-143
 @ ./src/components/ShortcutsModal.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Breakpoints/Breakpoints.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Breakpoints/Breakpoints.css 4:14-152
 @ ./src/components/SecondaryPanes/Breakpoints/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/Frames.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/Frames.css 4:14-147
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/Editor/Preview/Popup.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/Editor/Preview/Popup.css 4:14-146
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Popover.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Popover.css 4:14-142
 @ ./src/components/shared/Popover.js
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/WhyPaused.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/WhyPaused.css 4:14-150
 @ ./src/components/SecondaryPanes/Frames/WhyPaused.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/react-aria-components/src/tabs/tab-list.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/react-aria-components/src/tabs/tab-list.css 4:14-117
 @ ./node_modules/react-aria-components/src/tabs/tab-list.js
 @ ./node_modules/react-aria-components/src/tabs/index.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./node_modules/react-aria-components/src/tabs/tab.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./node_modules/react-aria-components/src/tabs/tab.css 4:14-112
 @ ./node_modules/react-aria-components/src/tabs/tab.js
 @ ./node_modules/react-aria-components/src/tabs/index.js
 @ ./src/components/PrimaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/SecondaryPanes/Frames/Group.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/SecondaryPanes/Frames/Group.css 4:14-146
 @ ./src/components/SecondaryPanes/Frames/Group.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/Badge.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/Badge.css 4:14-140
 @ ./src/components/shared/Badge.js
 @ ./src/components/SecondaryPanes/Frames/Group.js
 @ ./src/components/SecondaryPanes/Frames/index.js
 @ ./src/components/SecondaryPanes/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js

ERROR in ./node_modules/css-loader??ref--4-1!./node_modules/postcss-loader/lib!./src/components/shared/BracketArrow.css
Module build failed: Error: Cannot find module 'postcss-bidirection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at module.exports (/home/ash/Documents/projects/debugger.html/postcss.config.js:37:7)
    at config.load.then (/home/ash/Documents/projects/debugger.html/node_modules/postcss-load-config/src/index.js:62:18)
 @ ./src/components/shared/BracketArrow.css 4:14-147
 @ ./src/components/shared/BracketArrow.js
 @ ./src/components/shared/Popover.js
 @ ./src/components/Editor/Preview/Popup.js
 @ ./src/components/Editor/Preview/index.js
 @ ./src/components/Editor/index.js
 @ ./src/components/App.js
 @ ./src/utils/bootstrap.js
 @ ./src/client/index.js
 @ ./src/main.development.js
 @ multi ./src/main.development.js
webpack: Failed to compile.
```

--- a/postcss.config.js
+++ b/postcss.config.js
@@ -34,13 +34,11 @@ module.exports = ({ file, options, env }) => {
 
   return {
     plugins: [
-      require("postcss-bidirection"),
       require("autoprefixer")({
         browsers: ["last 2 Firefox versions", "last 2 Chrome versions"],
         flexbox: false,
         grid: false
       }),
-      require("postcss-class-namespace")(),
       mapUrl(mapUrlDevelopment)
     ]
   };

